### PR TITLE
fixed swagger file -v1

### PR DESCRIPTION
**Pull request by swmachan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 1]
```diff
@@ -540,6 +532,16 @@
...
               "$ref": "#/definitions/TranslateResultAll"
+            },
+            "headers": {
+              "E-Tag": {
+                "type": "string",
+                "description": "Current value of the entity tag for the requested groups of supported languages. To make subsequent requests more efficient, the client may send the ETag value in an If-None-Match header field."
+              },
+              "X-RequestId": {
+                "type": "integer",
+                "decription": "Value generated by the service to identify the request. It is used for troubleshooting purposes."
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]:
```suggestion
                "description": "Value generated by the service to identify the request. It is used for troubleshooting purposes."
```

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 2]
```diff
@@ -12,31 +12,23 @@
...
   },
   "securityDefinitions": {
-    "apim_key": {
+    "apiKeyHeader": {
```
Comment (yungezz) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]: this is breaking change, need a new service tag. Just to double confirm, is this a swagger file bug fixing, or service behavior change?

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 3]
```diff
@@ -540,6 +532,16 @@
             "description": "Success",
             "schema": {
               "$ref": "#/definitions/TranslateResultAll"
+            },
+            "headers": {
+              "E-Tag": {
+                "type": "string",
+                "description": "Current value of the entity tag for the requested groups of supported languages. To make subsequent requests more efficient, the client may send the ETag value in an If-None-Match header field."
+              },
+              "X-RequestId": {
```
Comment (yungezz) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]: header naming normally is small case.

**Comment by yungezz** [comment]
@swmachan pls fix CI failure.

**Comment by yungezz** [comment]
Hi @swmachan are you still on this PR? could you pls fix CR failure or close the PR if you aren't ready for review?

**Comment by swmachan** [comment]
Closing PR as I need to make fixes.

**Pull request closed without merging**